src/libstd/sys/unix/process/process_common.rs

```diff
@@ -375,8 +375,12 @@ impl ChildStdio {
 
 impl fmt::Debug for Command {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(f, "{:?}", self.program)?;
-        for arg in &self.args {
+        if self.program == self.args[0] {
+            write!(f, "{:?}", self.program)?;
+        } else {
+            write!(f, "[{:?}] {:?}", self.program, self.args[0])?;
+        }
```

Comment:
As a minor nit, could you common up the branches of this and avoid the else, like this:

```rust
if self.program != self.args[0] {
    write!(f, "[{:?}] ", self.program)?;
}
write!(f, "{:?}", self.args[0])?;
```

(It probably isn't worth trying to join this with the following loop and get the spaces right.)

Reply:
Done